### PR TITLE
Disable LegacyHitExplosion for hold notes

### LINKED ISSUE #9888: Mania long note sliderticks should not display lightingN
The sliderticks of long notes on osu!lazer display the full lightingN animation or image, which causes cluttered gameplay/confusion if lightingN has a large image or long animation. 
bc376321397f
version 2020.814.0

### PR DESCRIPTION
**Pull request by lanpai**
Resolves https://github.com/ppy/osu/issues/9888

### CODE REVIEW

File: osu.Game.Rulesets.Mania/Skinning/LegacyHitExplosion.cs [hunk 1]
```diff
@@ -66,10 +67,13 @@ private void onDirectionChanged(ValueChangedEvent<ScrollingDirection> direction)
 
         public void Animate(JudgementResult result)
         {
-            (explosion as IFramedAnimation)?.GotoFrame(0);
+            if (!(result.Judgement is HoldNoteTickJudgement))
```
Comment (bdach) [inline osu.Game.Rulesets.Mania/Skinning/LegacyHitExplosion.cs:70]: I would flip this check and early-return to reduce nesting.

Comment (lanpai) [reply osu.Game.Rulesets.Mania/Skinning/LegacyHitExplosion.cs]: Addressed in latest commit

**Review by peppy — DISMISSED** [review]

**Comment by bdach** [comment]
@lanpai Please pay attention to CI status going forward; your changes had a code style issue with spacing. We've fixed that for you in f6ca316, but it's easier for everyone involved if you keep track.

**Comment by peppy** [comment]
@smoogipoo can you double-check this is correct? not 100% sure on it.

**Comment by smoogipoo** [comment]
It seems fine. Stable doesn't have ticks.

**Review by bdach — APPROVED** [review]

**Pull request merged**